Missing some files into dist of v0.19.0

I've added Excalibur to an empty application Angular7 application, and the "npm start" say :

```
ERROR in ./node_modules/excalibur/dist/Loader.js
Module not found: Error: Can't resolve './Loader.css' in '[...]\node_modules\excalibur\dist'
ERROR in ./node_modules/excalibur/dist/Loader.js
Module not found: Error: Can't resolve './Loader.logo.png' in '[...]\node_modules\excalibur\dist'
```

I've added these files from the Github repo and it's works. So can you add these files into next version?

Greats!

--- a/.gitignore
+++ b/.gitignore
@@ -5,6 +5,7 @@
 /src/engine/*.map
 /sandbox/**/*.js
 /sandbox/**/*.d.ts
+/sandbox/lib/**
 /src/spec/*.js
 _SpecRunner.html
 TestsSpec.js

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
+- Fixed issue where there were missing files in the dist (Loader.css, Loader.logo.png) ([#1057](https://github.com/excaliburjs/Excalibur/issues/1057))
+
 <!--------------------------------- DO NOT EDIT BELOW THIS LINE --------------------------------->
 
 ## [0.19.0] - 2018-10-13

--- a/GruntFile.js
+++ b/GruntFile.js
@@ -103,6 +103,16 @@ module.exports = function(grunt) {
     // Copy dists for visual compilation/testing
     //
     copy: {
+      core: {
+        files: [
+          {
+            expand: true,
+            cwd: './src/engine/',
+            src: ['**/*.png', '**/*.css'],
+            dest: './build/dist/'
+          }
+        ]
+      },
       visual: {
         files: [
           {
@@ -290,6 +300,7 @@ module.exports = function(grunt) {
     'shell:gitBuild',
     'clean',
     'ts:core_es2015',
+    'copy:core',
     'string-replace',
     'webpack:bundle',
     'webpack:bundlemin',

